src/dos/dos_keyboard_layout.cpp

```diff
@@ -193,10 +215,11 @@ static void log_layout_read_error()
 	            strerror(errno));
 }
 
-static uint32_t read_kcl_file(const FILE_unique_ptr &kcl_file, const char *layout_id, bool first_id_only)
+static uint32_t read_kcl_file(const FILE_unique_ptr& kcl_file,
```

Comment:
What is `kcl`? I have no idea. Could you please spell this out in the name; it's not a common abbreviation.

Reply:
The FreeDOS keyboard layouts are specified in a human-readable specification files, then they are compiled to .kcl files using the KC compiler, and only then they are inked into the .SYS file as known from MS-DOS; the "compiler" and "linker" are available here: https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/dos/keyb/fdkeyb-2.0/

Unfortunately, I don't know yet what the KCL stands for, but I have added a comment telling what the KCL file is.